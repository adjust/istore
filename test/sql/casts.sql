SET DATESTYLE TO ISO;
SELECT '2014-06-12 16:31:40'::aj_time::date;
SELECT '2014-06-12 16:31:40'::aj_time::timestamp;
SELECT '2014-06-12 16:31:40'::timestamp::aj_time;
SELECT '2014-06-12'::aj_date::date;
SELECT '2014-06-12'::date::aj_date;
SELECT date('2014-06-12 16:31:40'::aj_time);