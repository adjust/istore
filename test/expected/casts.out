SET DATESTYLE TO ISO;
SELECT '2014-06-12 16:31:40'::aj_time::date;
    date    
------------
 2014-06-12
(1 row)

SELECT '2014-06-12 16:31:40'::aj_time::timestamp;
      timestamp      
---------------------
 2014-06-12 16:31:40
(1 row)

SELECT '2014-06-12 16:31:40'::timestamp::aj_time;
       aj_time       
---------------------
 2014-06-12 16:31:40
(1 row)

SELECT '2014-06-12'::aj_date::date;
    date    
------------
 2014-06-12
(1 row)

SELECT '2014-06-12'::date::aj_date;
  aj_date   
------------
 2014-06-12
(1 row)

SELECT date('2014-06-12 16:31:40'::aj_time);
    date    
------------
 2014-06-12
(1 row)

